go/vt/mysqlctl/backup.go

```diff
@@ -318,15 +319,14 @@ func Restore(ctx context.Context, params RestoreParams) (*BackupManifest, error)
 
 	// Add backupTime and restorePosition to LocalMetadata
 	params.LocalMetadata["RestoredBackupTime"] = manifest.BackupTime
-	// TODO(deepthi): Uncomment this when the table is fixed to fit replication positions.
-	//params.LocalMetadata["RestorePosition"] = mysql.EncodePosition(manifest.Position)
+	params.LocalMetadata["RestorePosition"] = mysql.EncodePosition(manifest.Position)
 
 	// Populate local_metadata before starting without --skip-networking,
 	// so it's there before we start announcing ourselves.
 	params.Logger.Infof("Restore: populating local_metadata")
 	err = PopulateMetadataTables(params.Mysqld, params.LocalMetadata, params.DbName)
 	if err != nil {
-		return nil, err
+		params.Logger.Errorf("Error populating metadata tables: %v, continuing..", err)
```

Comment:
If local_metadata fails, it could cause integrations like Orchestrator to exhibit unexpected behavior. Did we ever have PopulateMetadataTables optional like this before? Or was it always required?

Reply:
From the time it was introduced, an error populating metadata has always been fatal.

Reply:
I have reverted this change.